src/atoms/Switch/Switch.tsx

```diff
@@ -11,9 +12,13 @@ const LabelText = styled(Typography)`
   margin-top: 0.75em;
   cursor: pointer;
 `;
-export const Checkbox = styled(Input)`
+export const Checkbox = styled(Input)<{
+  greyable?: boolean;
+  checked?: boolean;
```

Comment:
I'm not sure if we need to add checked as a prop, since it's already on the `<input />` that this component renders. However TypeScript may not like that.

Reply:
Good call yeah. It doesn't need me to add it.

Reply:
Thanks. Do we still need this on Checkbox, Slider, and Props?